english/mystory.txt

```diff
@@ -0,0 +1,13 @@
+Once a cat was caught in a hunter's net. A mouse used to live in a nearby hole. 
+The mouse seeing the cat in the net started playing around the cat. 
+Soon a mongoose came there. He wanted to kill the mouse. As he lifted his head, he saw an owl sitting on a tree trying to catch him. The mongoose went very near the cat's net to save himself. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.